### DISCUSSION TITLE
Publish on Modrinth

### DISCUSSION THREAD

**Discussion by ModProg** [sketchmacaw/MacawsModsIssues#252] [Ideas]
Modrinth starts to emerge as a quite polished minecraft mod distribution platform, and they recently added add-revenue for mod publishers.

Did you consider to also publish there?

**Comment by sketchmacaw** [comment]
Yes, it will be published there soon. Just want to release some more bug fixes so the first versions there are as polished as then can be.

**Comment by GameBurrow** [comment]
Any updates on this? Your mods are one of the last mods in our modpack that aren't available in Modrinth.

**Comment by sketchmacaw** [comment]
Done, forgot about it
